dlt/destinations/impl/destination/__init__.py

```diff
@@ -1,16 +1,21 @@
 from typing import Optional
 from dlt.common.destination import DestinationCapabilitiesContext, TLoaderFileFormat
+from dlt.common.destination.capabilities import TLoaderParallelismStrategy
 
 
 def capabilities(
     preferred_loader_file_format: TLoaderFileFormat = "typed-jsonl",
     naming_convention: str = "direct",
     max_table_nesting: Optional[int] = 0,
+    max_parallel_load_jobs: Optional[int] = 0,
```

Comment:
I'm not sure we really need this here, this can be configured via config toml, same for the strategy. But maybe it is nice to have.

Reply:
it is really helpful. use case: qdrant in local mode must be sequential. (single thread). in #998 capabilities are adjusted per config values (and naming convention) so I'll be able to request a strategy based on it.